contribs/gmf/src/authentication/component.html

```diff
@@ -1,3 +1,8 @@
+<div ng-if="$ctrl.gmfUser.is_intranet">
+  <div class="form-group">
+    <span>{{'You are recognise as an intranet user.' | translate}}</span>
```

Comment:
You are recognized as an Intranet user.

Reply:
Intranet don't take a capital letter (in opposition of Internet), see e.-g.:
https://en.wikipedia.org/wiki/Intranet

Reply:
OK... but you still need a "d" at the end of "recognize".

Reply:
Already done ...